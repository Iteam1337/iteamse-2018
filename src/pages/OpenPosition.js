// @flow

import * as React from 'react'
import * as Iteam from '../typings/iteam.flow'
import * as IteamCMS from './__generated__/OpenPositionPage'
import { Query } from 'react-apollo'
import gql from 'graphql-tag'
import Header from '../components/Header/Header'
import Block from '../components/Blocks/Block'
import ImageBleed from '../components/Blocks/ImageBleed'

type Props = {
  match: {
    params: {
      id: string,
    },
  },
}

type QueryProps = Iteam.ApolloBase<IteamCMS.OpenPositionPage>

const OpenPositionPageQuery = gql`
  query OpenPositionPage($id: String!) {
    pageOpenPosition(id: $id) {
      headerImage
      headerText1
      headerText2
      headerTextBgColor
      application
      applicationTitle
      aboutUs
      aboutUsTitle
      bonusKnowledge
      bonusKnowledgeTitle
      contentImage
      knowledge
      knowledgeTitle
      location
      role
      roleTitle
      technicalities
      technicalitiesTitle
      title
    }
  }
`

const Work = ({ match }: Props) => {
  return (
    <Query query={OpenPositionPageQuery} variables={{ id: match.params.id }}>
      {({ loading, data: { pageOpenPosition } }: QueryProps) => {
        if (loading) {
          return null
        }

        return (
          <React.Fragment>
            <Header
              backgroundImage={pageOpenPosition.headerImage}
              messageBgColor={pageOpenPosition.headerTextBgColor}
              messageOne={pageOpenPosition.headerText1}
              messageTwo={pageOpenPosition.headerText2}
            />

            <Block title={pageOpenPosition.roleTitle}>
              {pageOpenPosition.role}
            </Block>

            <Block title={pageOpenPosition.knowledgeTitle}>
              {pageOpenPosition.knowledge}
            </Block>

            <Block title={pageOpenPosition.bonusKnowledgeTitle}>
              {pageOpenPosition.bonusKnowledge}
            </Block>

            <ImageBleed image={pageOpenPosition.contentImage} />

            <Block concrete title={pageOpenPosition.aboutUsTitle}>
              {pageOpenPosition.aboutUs}
            </Block>

            <Block title={pageOpenPosition.technicalitiesTitle}>
              {pageOpenPosition.technicalities}
            </Block>

            <Block title={pageOpenPosition.applicationTitle}>
              {pageOpenPosition.application}
            </Block>
          </React.Fragment>
        )
      }}
    </Query>
  )
}

export default Work
