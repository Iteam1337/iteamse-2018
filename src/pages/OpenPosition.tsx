import gql from 'graphql-tag'
import * as React from 'react'
import { Query } from 'react-apollo'
import Helmet from 'react-helmet-async'
import { RouteComponentProps, withRouter } from 'react-router-dom'
import { OpenPositionPageQuery } from '../../typings/iteamse'
import Block from '../components/Blocks/Block'
import ImageBleed from '../components/Blocks/ImageBleed'
import Breadcrumbs from '../components/Breadcrumbs/Breadcrumbs'
import GridColumn from '../components/Grid/GridColumn'
import Header from '../components/Header/Header'
import Team from '../components/Team/Team'

export const OPEN_POSITION_PAGE_QUERY = gql`
  query OpenPositionPage($id: String!) {
    pageOpenPosition(id: $id) {
      headerImage
      headerText1
      headerText2
      headerTextBgColor
      application
      applicationTitle
      aboutUs
      aboutUsTitle
      bonusKnowledge
      bonusKnowledgeTitle
      contactTitle
      contentImage
      knowledge
      knowledgeTitle
      location
      perks
      perksTitle
      role
      roleTitle
      team
      technicalities
      technicalitiesTitle
      title
    }
  }
`

class OpenPositionQuery extends Query<OpenPositionPageQuery> {}

export class OpenPosition extends React.Component<
  RouteComponentProps<{ id: string; match: string }>
> {
  render() {
    const { match } = this.props
    return (
      <OpenPositionQuery
        query={OPEN_POSITION_PAGE_QUERY}
        variables={{ id: match.params.id }}
      >
        {({ loading, data }) => {
          if (loading || !data) {
            return null
          }

          const { pageOpenPosition } = data

          return (
            <>
              <Helmet>
                <title>
                  Iteam | {pageOpenPosition.headerText1} | {
                    pageOpenPosition.headerText2
                  }
                </title>
                <meta
                  property="og:title"
                  content={`Iteam | ${pageOpenPosition.headerText1} | ${
                    pageOpenPosition.headerText2
                  }`}
                />
                <meta
                  property="twitter:title"
                  content={`Iteam | ${pageOpenPosition.headerText1} | ${
                    pageOpenPosition.headerText2
                  }`}
                />
                {pageOpenPosition.headerImage && (
                  <meta
                    property="og:image"
                    content={`https:${pageOpenPosition.headerImage}`}
                  />
                )}
              </Helmet>
              <Header
                backgroundImage={pageOpenPosition.headerImage}
                messageBgColor={pageOpenPosition.headerTextBgColor}
                messageOne={pageOpenPosition.headerText1}
                messageTwo={pageOpenPosition.headerText2}
              />

              <GridColumn>
                <Breadcrumbs title={pageOpenPosition.roleTitle} />

                <Block title={pageOpenPosition.roleTitle}>
                  {pageOpenPosition.role}
                </Block>

                <Block title={pageOpenPosition.knowledgeTitle}>
                  {pageOpenPosition.knowledge}
                </Block>

                <Block title={pageOpenPosition.bonusKnowledgeTitle}>
                  {pageOpenPosition.bonusKnowledge}
                </Block>

                <ImageBleed image={pageOpenPosition.contentImage} />

                <Block title={pageOpenPosition.aboutUsTitle}>
                  {pageOpenPosition.aboutUs}
                </Block>

                <Block title={pageOpenPosition.perksTitle}>
                  {pageOpenPosition.perks}
                </Block>

                <Block title={pageOpenPosition.technicalitiesTitle}>
                  {pageOpenPosition.technicalities}
                </Block>

                <Block title={pageOpenPosition.applicationTitle}>
                  {pageOpenPosition.application}
                </Block>
              </GridColumn>

              <Team
                bgColor="green"
                callToAction={pageOpenPosition.contactTitle}
                shortName={pageOpenPosition.team}
              />
            </>
          )
        }}
      </OpenPositionQuery>
    )
  }
}

export default withRouter(OpenPosition)
