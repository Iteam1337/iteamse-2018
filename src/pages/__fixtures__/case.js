export const workCase = {
  headerBgColor: '#1616B2',
  logo:
    '//images.ctfassets.net/rj4r6yfcesw5/4p0ptdLj9SaECekOI4c6C2/c2cdeab1cebc9650ebb0e7778357c231/AF_ordbild_vit.png',
  slug: 'arbetsformedlingen',
  thumbnailImage:
    '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
  tags: ['API', 'Docker', 'UX', 'React', 'Vue'],
  title: 'Arbetsförmedlingen',
  introduction:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  introductionTitle: 'En ny digital tjänst',
  process:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  processTitle: 'Process',
  development:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  developmentTitle: 'Utveckling',
  aboutCompany:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  aboutCompanyTitle: 'Arbetsförmedlingen',
  partners: '[Digitalbyrån](https://hej.com)',
  partnersTitle: 'Samarbetspartners',
  contact:
    '- [Hans Rollman](mailto:hans.rollman@iteam.se)\n- Johan Littorin, Telenor, 0734 41 72 85',
  contactTitle: 'Presskontakt',
  quote:
    '"We feel that our crew at Iteam are more colleagues than suppliers. Rather than specifying a tech spec to work on, we collaborate, carefully chiseling out functionality as we go along. This way of working has minimized waste and it has also taken us further than we ever imagined”',
  quotePerson: 'Anna Andersson, chef Arbetsförmedlingen',
  __typename: 'Case',
}
