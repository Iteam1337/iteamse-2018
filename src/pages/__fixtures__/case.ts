export const workCase = {
  __typename: 'Case',
  aboutCompany:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  aboutCompanyTitle: 'Arbetsförmedlingen',
  contact:
    '- [Hans Rollman](mailto:hans.rollman@iteam.se)\n- Johan Littorin, Telenor, 0734 41 72 85',
  contactTitle: 'Presskontakt',
  development:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  developmentTitle: 'Utveckling',
  headerBgColor: '#1616B2',
  introduction:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  introductionTitle: 'En ny digital tjänst',
  logo:
    '//images.ctfassets.net/rj4r6yfcesw5/4p0ptdLj9SaECekOI4c6C2/c2cdeab1cebc9650ebb0e7778357c231/AF_ordbild_vit.png',
  partners: '[Digitalbyrån](https://hej.com)',
  partnersTitle: 'Samarbetspartners',
  process:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  processTitle: 'Process',
  quote:
    '"We feel that our crew at Iteam are more colleagues than suppliers. Rather than specifying a tech spec to work on, we collaborate, carefully chiseling out functionality as we go along. This way of working has minimized waste and it has also taken us further than we ever imagined”',
  quotePerson: 'Anna Andersson, chef Arbetsförmedlingen',
  slug: 'arbetsformedlingen',
  tags: ['API', 'Docker', 'UX', 'React', 'Vue'],
  team: null,
  thumbnailImage:
    '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
  title: 'Arbetsförmedlingen',
}
