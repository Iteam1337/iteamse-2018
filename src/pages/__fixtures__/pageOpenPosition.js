export const pageOpenPosition = {
  headerImage:
    '//images.ctfassets.net/rj4r6yfcesw5/2VBOGPr664aWq4cEieYasY/7d824e86c44854559806e31e02c2e02a/juniorux.jpg',
  headerText1: 'UX Designer',
  headerText2: 'Stockholm',
  headerTextBgColor: '#668CFF',
  application:
    'Maila CV, personligt brev och beskrivning av ett case till: [work@iteam.se](mailto:work@iteam.se)',
  applicationTitle: 'Ansökan',
  aboutUs:
    'Vi digitaliserar företag och organisationer genom strategi, kod och kultur. Med hjälp av de tre bitarna skapar vi innovativa tekniska lösningar som skapar värde, gör något bra för samhället och låter oss ha roligt längs vägen.\n\nVåra kunder har spännande utmaningar både tekniskt och affärsmässigt, och stor potential att göra skillnad inom sina områden. Vi jobbar bland annat med Taxi Stockholm, Vimla, Läkare utan gränser och Arbetsförmedlingen.\n\nVi är ca 30 personer fördelade i Stockholm och Göteborg. Hos oss tar du av dig skorna vid dörren och känner dig som hemma. Vi lär oss ständigt nya saker av varandra - genom mentorskap, utbyte inom projekt, måndagsfrukostar, kodlunch eller under labbtimmarna. Vi har kul tillsammans och tar hand om varandra i vardagen, löparspåret, på yogamattan, spelkvällar, after works och vid andra speciella tillfällen. Vi arbetar aktivt för att sätta ihop team med så varierande och kompletterande bakgrund som möjligt – oavsett om det gäller kön, könsidentitet och -uttryck, etnisk tillhörighet, religion och trosuppfattning, funktionalitet, sexuell läggning, ålder, utbildning eller social bakgrund. Våra team är alltid korsfunktionella. Alltid agila. Alltid nära beställaren, deras anställda, användare och intressenter.',
  aboutUsTitle: 'Om oss',
  bonusKnowledge:
    '- Kvantitativa researchmetoder (t.ex. enkäter och användningsdata i Google Analytics)\n- Förståelse för den tekniska miljö dina lösningar kommer att implementeras i (t.ex. grundläggande HTML och CSS)\n- Utbildning inom interaktionsdesign, människa-datorinteraktion, beteendevetenskap, kognitionspsykologi, digital design eller närliggande område',
  bonusKnowledgeTitle: 'Plus i kanten',
  contentImage:
    '//images.ctfassets.net/rj4r6yfcesw5/62hgzE8G3ueMa2YSEuG4og/db2b643db6041f9d26936bc2d6006607/bleed_howwework.jpg',
  knowledge:
    '- Kvalitativa researchmetoder (t.ex. användbarhetstester, observationer, intervjuer och workshops)\n- Kommunicera dina insikter med hjälp av lämpliga leverabler (t.ex. rapporter, användarresor, prototyper eller skrynkliga pappersskisser)\n- Interaktionsdesign\n- Vara användarens representant i projekt och stötta både kollegor och kunder med användarens perspektiv\n- Jobba agilt\n- Arbeta korsfunktionellt och förstå behoven hos kollegor ur andra yrkesgrupper\n- Goda kunskaper i engelska (för att kunna kommincera med dina kollegor)\n- Goda kunskaper i svenska (för att kunna genomföra användarstudier i kontexter där det primära användarspråket är svenska)',
  knowledgeTitle: 'Du behöver kunna',
  location: 'Stockholm',
  role:
    'Som UX designer på Iteam är din roll att hjälpa våra kunder med digitala lösningar som skapar värde på riktigt. Som gör något bra både för kunden, slutanvändaren och samhället. Med hjälp av olika researchmetoder undersöker du både användarens och kundens behov och kontext, samt definierar och testar användarupplevelser.\n\nDu delar med dig av din kunskap inom user experience. Du förstår också att om lösningar ska vara verkligt användarvänliga behöver även arbetsprocesserna och kulturen vara användarcentrerad. Vi strävar mot att UX aldrig ska få vara bara en fas eller en eftertanke. Det ska genomsyra hela projektet. Alltid. ',
  roleTitle: 'Din roll',
  technicalities:
    '__Några av våra kunder:__ Taxi Stockholm, Vimla, Swedbank, Läkare Utan Gränser, Arbetsförmedlningen, Alcro Beckers, Scania, SoundIdeas.\n\n__Anställningsform:__ Tillsvidareanställning. 6 månaders provanställning, sedan fast anställning.\n\n__Plats:__ Stockholm.\n\n__Start:__ Omgående.',
  technicalitiesTitle: 'Teknikaliteter',
  title: 'Junior UX Designer',
  __typename: 'OpenPosition',
}
