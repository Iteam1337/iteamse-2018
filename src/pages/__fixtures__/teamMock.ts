import { TEAM_QUERY } from '../../components/Team/Team'
import { teamMembers } from './teamMembers'

export const TeamQueryMock = {
  request: {
    query: TEAM_QUERY,
  },
  result: {
    data: {
      teamMembers,
    },
  },
}
