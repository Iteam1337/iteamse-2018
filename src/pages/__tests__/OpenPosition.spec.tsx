import React from 'react'
import { render, wait } from 'react-testing-library'
import MockedQuery from '../../utils/test-utils/MockedQuery'
import { pageOpenPosition } from '../__fixtures__/pageOpenPosition'
import { TeamQueryMock } from '../__fixtures__/teamMock'
import OpenPosition, { OPEN_POSITION_PAGE_QUERY } from '../OpenPosition'

const location = {
  pathname: '/jobba-hos-oss/junior-ux-designer',
}

const params = {
  id: 'junior-ux-designer',
}

const mocks = [
  {
    request: {
      query: OPEN_POSITION_PAGE_QUERY,
      variables: params,
    },
    result: {
      data: {
        pageOpenPosition,
      },
    },
  },
  TeamQueryMock,
]

describe('components/OpenPosition', () => {
  it('renders OpenPosition', async () => {
    const { container } = render(
      <MockedQuery location={location} mocks={mocks} params={params}>
        <OpenPosition />
      </MockedQuery>
    )

    await wait()

    expect(container).toMatchSnapshot()
  })
})
