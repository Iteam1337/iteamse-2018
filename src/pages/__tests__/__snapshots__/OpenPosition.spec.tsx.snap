// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/OpenPosition renders OpenPosition 1`] = `
<div>
  <div>
    <div
      class="sc-dqBHgY dyvxCo"
    >
      <div
        class="sc-gxMtzJ eXQJvD"
      >
        <div
          class="sc-btzYZH cZSewS"
        >
          <a
            class="sc-lhVmIH ikqhWa"
          >
            <img
              class="sc-bYSBpT dCPoBO"
              data-testid="logo"
              path="test-file-stub"
            />
          </a>
          <div
            class="sc-elJkPf kVhaog"
          >
            <a
              class="sc-kTUwUJ fEMxFB"
              aria-current="false"
            >
              Erbjudanden
            </a>
            <a
              class="sc-kTUwUJ fEMxFB"
              aria-current="false"
            >
              Case
            </a>
            <a
              class="sc-kTUwUJ fEMxFB"
              aria-current="false"
            >
              Metod
            </a>
            <a
              class="sc-kTUwUJ fEMxFB"
              aria-current="false"
            >
              Medarbetare
            </a>
            <a
              class="sc-kTUwUJ fEMxFB active-nav"
              aria-current="true"
            >
              Karriär
            </a>
            <a
              class="sc-kTUwUJ fEMxFB"
              aria-current="false"
            >
              Om
            </a>
            <div
              class="sc-jtRfpW hwCVJp"
            />
          </div>
        </div>
        <div
          class="sc-dfVpRl hlzXex"
        >
          <h1
            class="sc-gzOgki gWEEDl"
          >
            <span>
              UX Designer
            </span>
          </h1>
          <h1
            class="sc-gzOgki gWEEDl"
          >
            <span>
              Stockholm
            </span>
          </h1>
        </div>
      </div>
    </div>
  </div>
  <div
    class="sc-iwsKbI fwFOwW"
  >
    <div
      class="sc-gzVnrw cLTlOm"
      data-test="breadcrumbs"
    >
      <a
        class="sc-htoDjs dEAmle"
      >
        Jobba hos oss
      </a>
      <span>
        Din roll
      </span>
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="sc-bdVaJa fGzlqx"
        data-test="block-"
      >
        <div
          class="sc-bxivhb kgeHJJ"
        >
          <div
            class="sc-ifAKCX cFlEyZ"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              Din roll
            </h3>
          </div>
          <div
            class="sc-bZQynM hmvJjY"
          >
            <div
              class="sc-htpNat iEyarI"
            >
              <div>
                <p>
                  Som UX designer på Iteam är din roll att hjälpa våra kunder med digitala lösningar som skapar värde på riktigt. Som gör något bra både för kunden, slutanvändaren och samhället. Med hjälp av olika researchmetoder undersöker du både användarens och kundens behov och kontext, samt definierar och testar användarupplevelser.
                </p>
                <p>
                  Du delar med dig av din kunskap inom user experience. Du förstår också att om lösningar ska vara verkligt användarvänliga behöver även arbetsprocesserna och kulturen vara användarcentrerad. Vi strävar mot att UX aldrig ska få vara bara en fas eller en eftertanke. Det ska genomsyra hela projektet. Alltid. 
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="sc-bdVaJa fGzlqx"
        data-test="block-"
      >
        <div
          class="sc-bxivhb kgeHJJ"
        >
          <div
            class="sc-ifAKCX cFlEyZ"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              Du behöver kunna
            </h3>
          </div>
          <div
            class="sc-bZQynM hmvJjY"
          >
            <div
              class="sc-htpNat iEyarI"
            >
              <div>
                <ul>
                  <li>
                    Kvalitativa researchmetoder (t.ex. användbarhetstester, observationer, intervjuer och workshops)
                  </li>
                  <li>
                    Kommunicera dina insikter med hjälp av lämpliga leverabler (t.ex. rapporter, användarresor, prototyper eller skrynkliga pappersskisser)
                  </li>
                  <li>
                    Interaktionsdesign
                  </li>
                  <li>
                    Vara användarens representant i projekt och stötta både kollegor och kunder med användarens perspektiv
                  </li>
                  <li>
                    Jobba agilt
                  </li>
                  <li>
                    Arbeta korsfunktionellt och förstå behoven hos kollegor ur andra yrkesgrupper
                  </li>
                  <li>
                    Goda kunskaper i engelska (för att kunna kommincera med dina kollegor)
                  </li>
                  <li>
                    Goda kunskaper i svenska (för att kunna genomföra användarstudier i kontexter där det primära användarspråket är svenska)
                  </li>
                </ul>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="sc-bdVaJa fGzlqx"
        data-test="block-"
      >
        <div
          class="sc-bxivhb kgeHJJ"
        >
          <div
            class="sc-ifAKCX cFlEyZ"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              Plus i kanten
            </h3>
          </div>
          <div
            class="sc-bZQynM hmvJjY"
          >
            <div
              class="sc-htpNat iEyarI"
            >
              <div>
                <ul>
                  <li>
                    Kvantitativa researchmetoder (t.ex. enkäter och användningsdata i Google Analytics)
                  </li>
                  <li>
                    Förståelse för den tekniska miljö dina lösningar kommer att implementeras i (t.ex. grundläggande HTML och CSS)
                  </li>
                  <li>
                    Utbildning inom interaktionsdesign, människa-datorinteraktion, beteendevetenskap, kognitionspsykologi, digital design eller närliggande område
                  </li>
                </ul>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <img
      class="sc-gqjmRU kaGJTv"
      src="//images.ctfassets.net/rj4r6yfcesw5/62hgzE8G3ueMa2YSEuG4og/db2b643db6041f9d26936bc2d6006607/bleed_howwework.jpg"
    />
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="sc-bdVaJa fGzlqx"
        data-test="block-"
      >
        <div
          class="sc-bxivhb kgeHJJ"
        >
          <div
            class="sc-ifAKCX cFlEyZ"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              Om oss
            </h3>
          </div>
          <div
            class="sc-bZQynM hmvJjY"
          >
            <div
              class="sc-htpNat iEyarI"
            >
              <div>
                <p>
                  Vi digitaliserar företag och organisationer genom strategi, kod och kultur. Med hjälp av de tre bitarna skapar vi innovativa tekniska lösningar som skapar värde, gör något bra för samhället och låter oss ha roligt längs vägen.
                </p>
                <p>
                  Våra kunder har spännande utmaningar både tekniskt och affärsmässigt, och stor potential att göra skillnad inom sina områden. Vi jobbar bland annat med Taxi Stockholm, Vimla, Läkare utan gränser och Arbetsförmedlingen.
                </p>
                <p>
                  Vi är ca 30 personer fördelade i Stockholm och Göteborg. Hos oss tar du av dig skorna vid dörren och känner dig som hemma. Vi lär oss ständigt nya saker av varandra - genom mentorskap, utbyte inom projekt, måndagsfrukostar, kodlunch eller under labbtimmarna. Vi har kul tillsammans och tar hand om varandra i vardagen, löparspåret, på yogamattan, spelkvällar, after works och vid andra speciella tillfällen. Vi arbetar aktivt för att sätta ihop team med så varierande och kompletterande bakgrund som möjligt – oavsett om det gäller kön, könsidentitet och -uttryck, etnisk tillhörighet, religion och trosuppfattning, funktionalitet, sexuell läggning, ålder, utbildning eller social bakgrund. Våra team är alltid korsfunktionella. Alltid agila. Alltid nära beställaren, deras anställda, användare och intressenter.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="sc-bdVaJa fGzlqx"
        data-test="block-"
      >
        <div
          class="sc-bxivhb kgeHJJ"
        >
          <div
            class="sc-ifAKCX cFlEyZ"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              Några av våra bonusar
            </h3>
          </div>
          <div
            class="sc-bZQynM hmvJjY"
          >
            <div
              class="sc-htpNat iEyarI"
            >
              <div>
                <ul>
                  <li>
                    6 veckors semester
                  </li>
                  <li>
                    Kodluncher varje fredag för gemensamt lärande inkl lunch
                  </li>
                  <li>
                    Ingen förväntad övertid
                  </li>
                  <li>
                    Förmånligt friskvårdsbidrag och fokus på välmående
                  </li>
                  <li>
                    Förmånligt pensionssystem
                  </li>
                  <li>
                    Bonus vid slutet av ett bra år
                  </li>
                  <li>
                    Mentorskap mellan arbetskamrater
                  </li>
                  <li>
                    Individuell budget för kompetensutveckling
                  </li>
                </ul>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="sc-bdVaJa fGzlqx"
        data-test="block-"
      >
        <div
          class="sc-bxivhb kgeHJJ"
        >
          <div
            class="sc-ifAKCX cFlEyZ"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              Teknikaliteter
            </h3>
          </div>
          <div
            class="sc-bZQynM hmvJjY"
          >
            <div
              class="sc-htpNat iEyarI"
            >
              <div>
                <p>
                  <strong>
                    Några av våra kunder:
                  </strong>
                  Taxi Stockholm, Vimla, Swedbank, Läkare Utan Gränser, Arbetsförmedlningen, Alcro Beckers, Scania, SoundIdeas.
                </p>
                <p>
                  <strong>
                    Anställningsform:
                  </strong>
                  Tillsvidareanställning. 6 månaders provanställning, sedan fast anställning.
                </p>
                <p>
                  <strong>
                    Plats:
                  </strong>
                  Stockholm.
                </p>
                <p>
                  <strong>
                    Start:
                  </strong>
                  Omgående.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="sc-bdVaJa fGzlqx"
        data-test="block-"
      >
        <div
          class="sc-bxivhb kgeHJJ"
        >
          <div
            class="sc-ifAKCX cFlEyZ"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              Ansökan
            </h3>
          </div>
          <div
            class="sc-bZQynM hmvJjY"
          >
            <div
              class="sc-htpNat iEyarI"
            >
              <div>
                <p>
                  Maila CV, personligt brev och beskrivning av ett case till: 
                  <a
                    href="mailto:work@iteam.se"
                  >
                    work@iteam.se
                  </a>
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
